### PR TITLE
add offline-cloud svg icon

### PR DESCRIPTION
**Pull request by joecowton1**
## What is the purpose of this change?

We require an offline cloud icon for a placeholder card in Editions if a video is unavailable because the user is offline.

https://www.figma.com/file/BnRUbmmtqZREAcJmHGBCq4/video-not-available?node-id=5%3A164

## What does this change?

-   adds offline cloud icon

### Accessibility

-   [ ] [Tested with screen reader](https://www.theguardian.design/2a1e5182b/p/6691bb-accessibility/t/78ac51)
-   [ ] [Navigable with keyboard](https://www.theguardian.design/2a1e5182b/p/6691bb-accessibility/t/009027)
-   [ ] [Colour contrast passed](https://www.theguardian.design/2a1e5182b/p/6691bb-accessibility/t/58dbf2)
-   [ ] [The component doesn't use only colour to convey meaning](https://www.theguardian.design/2a1e5182b/p/6691bb-accessibility/t/61524d)

### Cross browser and device testing

-   [ ] Tested with touch screen device

### Responsiveness

-   [ ] Tested at all breakpoints
-   [ ] Tested with with long text
-   [ ] Stretched to fill a wide container

### Documentation

-   [ ] Full API surface area is documented in the README
-   [x] Examples in Storybook

### CODE REVIEW

**Review by mxdvl — CHANGES_REQUESTED** [review]

**Comment by SiAdcock** [comment]
Thanks both! I’m guessing this build fail can be fixed by running “yarn fix” and committing the resulting changes. Would you like to give it a try?

File: src/core/icons/offline-cloud.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const SvgOfflineCloud = () => (
+	<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 33">
+		<path d="M29.9 12.64a7.27 7.27 0 112.82 14H20.95c-1.12.05-5.43.04-8.92.03l2.75-2.76h17.96a4.55 4.55 0 10-3.64-7.27 5.68 5.68 0 00-3.37-3.67l2.05-2.06c.8.45 1.52 1.03 2.14 1.73zM25.18 9.96l8.2-8.2L31.61 0l-8.8 8.8A9.39 9.39 0 0014.26 3a9.6 9.6 0 00-9.63 9.55C1.94 13.74 0 16.5 0 19.36a7.29 7.29 0 005.26 7L.5 31.1l1.77 1.77 6.23-6.23 14.1-14.1 2.58-2.59zm-10.9-4.23c3.1 0 5.56 2.34 6.36 5.25L7.7 23.9h-.43a4.52 4.52 0 01-4.54-4.55c0-2.08 1.68-4 3.63-4.54l.82 1.82 1.1-.37-.06-.15a10.2 10.2 0 01-.86-3.57 6.86 6.86 0 016.91-6.82z"/>
```
Comment (mxdvl) [inline src/core/icons/offline-cloud.tsx:3]:
```suggestion
		<path d="M29.9 12.64a7.27 7.27 0 112.82 14H20.95c-1.12.05-5.43.04-8.92.03l2.75-2.76h17.96a4.55 4.55 0 10-3.64-7.27 5.68 5.68 0 00-3.37-3.67l2.05-2.06c.8.45 1.52 1.03 2.14 1.73zM25.18 9.96l8.2-8.2L31.61 0l-8.8 8.8A9.39 9.39 0 0014.26 3a9.6 9.6 0 00-9.63 9.55C1.94 13.74 0 16.5 0 19.36a7.29 7.29 0 005.26 7L.5 31.1l1.77 1.77 6.23-6.23 14.1-14.1 2.58-2.59zm-10.9-4.23c3.1 0 5.56 2.34 6.36 5.25L7.7 23.9h-.43a4.52 4.52 0 01-4.54-4.55c0-2.08 1.68-4 3.63-4.54l.82 1.82 1.1-.37-.06-.15a10.2 10.2 0 01-.86-3.57 6.86 6.86 0 016.91-6.82z" />
```

**Review by mxdvl — APPROVED** [review]
Sorry my suggestion wasn’t formatted properly. All good from an SVG perspective 📐

**Comment by joecowton1** [comment]
Thanks @mxdvl .

@SiAdcock have squashed those commits now, could we possibly get this merged and do a little release this week?

**Review by SiAdcock — APPROVED** [review]

**Pull request merged**